bin/player.rb

```diff
@@ -1,9 +1,84 @@
-	class Player
-        attr_reader :name, :symbol
 
-        def initialize(name, symbol)
-            @name = name
-            @symbol = symbol
-        end
+
+class Player
+  attr_reader :name, :symbol
+
+  def initialize(name, symbol)
+    @name = name
+    @symbol = symbol
+  end
+end
+
```

Comment:
Please for clearly see OOP in action, I suggest you put each class in its own file